### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by stephenwaite**
bug fix

### CODE REVIEW

File: interface/forms/fee_sheet/review/fee_sheet_ajax.php [hunk 1]
```diff
@@ -82,7 +87,16 @@
     }
 
     foreach ($json_procs as $proc) {
-        $procs[] = new procedure($proc->{'code'}, $proc->{'code_type'}, $proc->{'description'}, $proc->{'fee'}, $proc->{'justify'}, $proc->{'modifiers'}, $proc->{'units'}, 0);
+        $procs[] = new procedure(
+            $proc->{'code'},
```
Comment (adunsulag) [inline interface/forms/fee_sheet/review/fee_sheet_ajax.php:91]: I really dislike curly brace syntax like this :( Not sure why we still need to use it.

**Comment by adunsulag** [comment]
Haven't tested the code but it looks fine to me from a code review standpoint.  Although I really don't like the curly brace syntax but its existing code so oh well...

File: interface/forms/fee_sheet/review/fee_sheet_ajax.php (hunk 1 above)
Comment (devspacenine) [reply interface/forms/fee_sheet/review/fee_sheet_ajax.php:91]: Are there any good php linters or prettierjs-esque  packages that would remove the superfluous curly braces?

Comment (stephenwaite) [reply interface/forms/fee_sheet/review/fee_sheet_ajax.php:91]: we use squizlabs/php_codesniffer, https://github.com/openemr/openemr/blob/master/CONTRIBUTING.md#dev_tools_tests, which isn't picking this up

**Comment by stephenwaite** [comment]
testing well

**Review by bradymiller — APPROVED** [review]

**Comment by bradymiller** [comment]
https://user-images.githubusercontent.com/278968/187527383-3aab0e4a-dd4e-42f1-b9a7-dae420f5966d.mp4

**Pull request merged**